instance/server/Client/Admin/Views.elm

```diff
@@ -11,12 +11,12 @@ import Client.Components exposing (..)
 import Client.Styles exposing (..)
 import Shared.Test exposing (..)
 import Shared.User exposing (..)
-import Shared.Routes exposing (..)
+import Shared.Routes exposing (Route(..), toPath, assets)
 
 
 loginView =
     form
-        [ action routes.login
+        [ action (toPath Login)
```

Comment:
That's true, if you are expecting it to not be your standard API. Here, we're expecting _everything_ to be handled through this API, so helpers are better and make the code more readable